lib/inc/facter/facts/fact.hpp

```diff
@@ -434,11 +434,15 @@ namespace facter { namespace facts {
         /**
          * The fact for virtualization hypervisor.
          */
-        constexpr static char const* virtualization = "virtual";
+        constexpr static char const* virt = "virtual";
```

Comment:
Any reason to still change `virtualization` to `virt`? The old one doesn't seem to conflict with anything, and this is still a public header file.